agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/templates/notification/mail/EventMailPlugin.gtmpl

```diff
@@ -213,6 +213,7 @@
                                             <td style="color:#999999; font-size:13px;font-family:HelveticaNeue,arial,tahoma,serif; font-weight:bold"><%=_ctx.appRes("Notification.agenda.event.attendees")%></td>
                                             <td align="left"><%= attendees %></td>
                                         </tr>
+                                        <% try { %>
```

Comment:
Could we instead of addiong try/catch add the property to the list of properties in the notification  object ?